server/service/src/repack/generate.rs

```diff
@@ -25,13 +25,13 @@ struct StockLineJob {
 
 pub fn generate(
     ctx: &ServiceContext,
-    stock_line: StockLineRow,
+    stock_line: StockLine,
```

Comment:
Is this related to what you mentioned the other day, that StockLineRow and its ilk should be private?

Reply:
The row structs are really just the Diesel structs to deserialize from the database. We used these row objects as domain objects to avoid doing a 1 to 1 mapping from Diesel object to our Domain object (StockLineRow -> StockLine). This seemed very laborious and cumbersome at the time... Now this would have come in handy though and would have saved us some time.

We should at least have used a type alias consistently `type StockLine = StockLineRow`...

So yes the Diesel structs should best be private.